php/blocks/class-loader.php

```diff
@@ -394,7 +394,7 @@ public function block_template( $name, $type = 'block' ) {
 
 		// So lets fix it.
 		if ( empty( $wp_query ) ) {
-			$wp_query = new \WP_Query(); // phpcs: ignore WordPress.WP.GlobalVariablesOverride.Prohibited.
+			$wp_query = new \WP_Query(); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
```

Comment:
Weird that this wasn't picked up in my version of phpcs. I'll update.

Reply:
All good, I was having a lot of trouble running PHPCS locally.